src/c++/perf_analyzer/request_rate_worker.cc

```diff
@@ -62,6 +63,19 @@ RequestRateWorker::CreateContexts()
   while (ctxs_.size() < active_ctx_cnt) {
     CreateContext();
   }
+
+  ResetFreeCtxIds();
+}
+
+void
+RequestRateWorker::ResetFreeCtxIds()
```

Comment:
2 things: 
1. this method exists on the concurrency side but has special behavior for sequence models. Is that necessary here?
2. If it is not necessary, can this be pushed to the load_manager as well?